addons/mail/static/src/models/user_setting.js

```diff
@@ -132,9 +131,9 @@ registerModel({
             await this.messaging.rtc.updateVoiceActivation();
         },
         async togglePushToTalk() {
-            this.update({ use_push_to_talk: !this.use_push_to_talk });
+            this.update({ isPushToTalkUsed: !this.isPushToTalkUsed });
             await this.messaging.rtc.updateVoiceActivation();
-            if (!this.messaging.isCurrentUserGuest) {
+            if (this.messaging.currentUser) {
```

Comment:
4th commit: this change from `!this.messaging.isCurrentUserGuest` to `this.messaging.currentUser` can be in its own commit